libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -1299,19 +1299,20 @@ public function testAllowedDestinationConfigurations(
     #[NeedsEmptyOutputBucket]
     public function testWriteAlwaysFlag(): void
     {
-        if ($this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
-            self::markTestSkipped('Write always flag is not implemented for slice feature.');
-        }
+        $root = $this->temp->getTmpFolder() . '/upload/';
+        file_put_contents($root . 'table0.csv', "\"Id\",\"Name\"\n\"test\",\"test\"\n");
 
         $this->testAllowedDestinationConfigurations(
             null,
             null,
             [
+
                 [
-                    'source' => 'table.csv',
+                    'source' => 'table0.csv',
```

Comment:
Same source tam byl jen kvuli tomu ze ten `testAllowedDestinationConfigurations` zaklada fyzicky jeden source.

Ono to jen overuje situaci, kdy se posle write table s validnim mappingem a isFailedJob flagem. A zapise se jen to co ma `write_always`.